_meta/beat.yml

```diff
@@ -183,6 +183,56 @@ apm-server:
   #ilm:
     #enabled: false
 
+  #kibana:
```

Comment:
has there been a discussion around using the existing kibana settings instead of introducing a new one?

Reply:
Yes, it has been discussed.

This is the same setting, just in a different namespace. 

The beats setting is under `setup` which is not correct in our case; this is not for a setup command.

Also, should we use that, the server would refuse to start if Kibana is not reachable, which is not what we want, and not coherent eg. with Elasticsearch output.

Reply:
Gotcha 👍